tools/release_provenance/test_release_data.json

```diff
@@ -0,0 +1,16 @@
+{
+    "spack_hash": "ewcdbrfukblyjxpkhd3mfkj4yxfolal4",
+    "spec":  "mom5@git.2023.11.09=2023.11.09",
+    "model_build": {
+        "spack_hash": "f4f2qe5b3c22q4dtzzqs5o2iygy435re",
+        "spec": "access-om2@git.2023.11.23=2023.11.23",
```

Comment:
How does this bit work? Is this the spec/spack_hash of the model that contains the components? What happens if we need an entry for `access-om3` or `access-om2` rather than the components?